openpype/hosts/fusion/deploy/MenuScripts/install_pyside2.py

```diff
@@ -0,0 +1,14 @@
+# This is just a quick hack for users running Py3 locally but having no
+# Qt library installed
+import os
+
+try:
+    print("Qt library found, nothing to do.")
+    from Qt import QtWidgets
+except ImportError as exc:
+    print("Assuming no Qt library is installed..")
+    print('Installing PySide2 for Python 3.6: '
+          f'{os.environ["FUSION16_PYTHON36_HOME"]}')
+
+    import subprocess
+    subprocess.Popen(["pip", "install", "PySide2"])
```

Comment:
**Warning:** This can install into e.g. OpenPype `.venv` due to not explicitly enough installing into the correct python.

A fix was implemented here: https://github.com/BigRoy/OpenPype/commit/3d5fd09dcbd0b7e10dd63595f8f78475194be15b